Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM quay.io/fedora/fedora:34-x86_64 as builder
 LABEL stage=builder
 WORKDIR /build
 
-# install dependencies and go 1.16
+# install dependencies and go 1.17
```

Comment:
Any reason for we are not using 1.18 yet?

Reply:
We probably should move to 1.18 here - though I think it should be done in another PR